.github/workflows/ci-build.yaml

```diff
@@ -173,7 +173,9 @@ jobs:
           sudo find /usr/local/include -type f | xargs sudo chmod a+r
           sudo find /usr/local/include -type d | xargs sudo chmod a+rx
           ls /usr/local/include/google/protobuf/
-      - run: docker pull openapitools/openapi-generator-cli:v5.2.1 &
+      - run: |
+          docker pull openapitools/openapi-generator-cli:v5.4.0 &
+          docker pull openapitools/openapi-generator-cli:v5.2.1 &
```

Comment:
Why are we pulling two versions?

Reply:
The python gen is upgraded to v5.4.0
Java gen is facing some issues in v5.4.0, unchanged for now, remain at 5.2.1